modules/auth/auth_form.go

```diff
@@ -24,11 +24,13 @@ type AuthenticationForm struct {
 	AttributeName                 string
 	AttributeSurname              string
 	AttributeMail                 string
+	AttributeSSHPublicKey         string
 	AttributesInBind              bool
 	Filter                        string
 	AdminFilter                   string
 	IsActive                      bool
 	IsSyncEnabled                 bool
+	IsLdapPublicSSHKeySyncEnabled bool
```

Comment:
Ok, I've thought about this for some more time and I agree with you that it's really not needed. Will delete it.